src/Data/Quantity.hs

```diff
@@ -128,3 +129,6 @@ mkPercentage i
 data MkPercentageError
     = PercentageOutOfBoundsError
     deriving (Show, Eq)
+
+zeroPercentage :: Percentage
+zeroPercentage = Percentage 0
```

Comment:
Added this as there is no Percentage constructor being exported from that module and needed something to compare restoration progress in integration tests (`expectFieldEqual state (Restoring (Quantity zeroPercentage)) r`). (Probably in practice we will be comparing that restoration is finished or Percantage different from zero)

Reply:
`Percentage` has a `Bounded` and `Enum` instances, so you can just use `minBound` or `toEnum 0` for this ;)